Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/validation/validation.go

```diff
@@ -419,6 +419,10 @@ func ValidateGeneratedSelector(obj *extensions.Job) field.ErrorList {
 	allErrs = append(allErrs, apivalidation.ValidateHasLabel(obj.Spec.Template.ObjectMeta, field.NewPath("spec").Child("template").Child("metadata"), "controller-uid", string(obj.UID))...)
 	allErrs = append(allErrs, apivalidation.ValidateHasLabel(obj.Spec.Template.ObjectMeta, field.NewPath("spec").Child("template").Child("metadata"), "job-name", string(obj.Name))...)
 	expectedLabels := make(map[string]string)
+	// Expect all the labels that are actually on the template
```

Comment:
Sure, waiting to hear how I'm supposed to actually use the API to accomplish what we used to do